docs/docs/100-reference/01-command-line/acorn_install.md

```diff
@@ -47,6 +47,7 @@ acorn install
       --features strings                                  Enable or disable features. (example foo=true,bar=false)
   -h, --help                                              help for install
       --http-endpoint-pattern string                      Go template for formatting application http endpoints. Valid variables to use are: App, Container, Namespace, Hash and ClusterDomain. (default pattern is {{hashConcat 8 .Container .App .Namespace | truncate}}.{{.ClusterDomain}})
+      --ignore-resource-request                           Ignore memory and CPU requests and limits, intended for local development (default is false)
```

Comment:
flag name may not be right...

Reply:
What we are doing is ignoring the memory and other resource requirements as defined by the compute class and the app. Maybe a better name is
```
--ignore-resource-requirements
```
...in fact, the function where we check this flag is called `ResourceRequirements`, so that I kind of aligns better.

Reply:
It also isn't just resource requests. It is limits as well